plugins/woocommerce-admin/client/two-column-tasks/style.scss

```diff
@@ -109,6 +109,10 @@
 	margin: 0 auto;
 	justify-content: space-between;
 
+	&.woocommerce-task-list__setup_experiment_2 {
+		margin-bottom: 16px;
```

Comment:
Super nitpicky but we can use `$gap` for `16px`

Reply:
Thanks @ilyasfoo, updated in 6d0b794fc7ae864316a090789b71f4b0903476f4. 🙂